src/OrbitBase/FileTest.cpp

```diff
@@ -305,6 +305,22 @@ TEST(File, MoveFile) {
   EXPECT_FALSE(exists_or_error.value());
 }
 
+TEST(File, RemoveFile) {
+  auto tmp_file_or_error = TemporaryFile::Create();
+  ASSERT_THAT(tmp_file_or_error, HasNoError());
+  auto tmp_file = std::move(tmp_file_or_error.value());
+
+  // Windows is weird, for some reason it does not allow to move opened files.
```

Comment:
This comment is from the `MoveFile` test, isn't it? It makes no sense here

Reply:
fixed comment.